tools/api_versioning/generate_api_version_header_test.py

```diff
@@ -1,7 +1,8 @@
 """Tests the api version header file generation.
 """
 import generate_api_version_header
-from generate_api_version_header import ApiVersion
+import utils
+from utils import ApiVersion
```

Comment:
I think the `import utils` is redundant here and above.

Reply:
Done.